src/app/cli/src/init/client.ml

```diff
@@ -943,25 +943,40 @@ let snark_job_list =
          | Error e ->
              print_rpc_error e ))
 
+let json_request ~summary ~graphql ~name input_flags =
```

Comment:
I feel slightly reluctant to reimplement part of the graphql_ppx parsing logic unless we're seeing an actual perf issue? Just adds more surface area for needing to handle parse errors.

The `query_raw` also seems to bind our output format to the graphql query format.

Reply:
So, I tried to parse the json response to it's actual ocaml type and then print out its JSON representation. However, I encountered a problem. Namely, I could not convert the json representation into an OCaml type as I would need to create a record or an object with the field `to`. `to` is a reserved keyword in OCaml.

After encountering this, I think it's just better to print out the raw json and not even parse the raw json. I don't think there would be many edge cases to take care of if we query with `query_raw`.